src/common/checksum.c

```diff
@@ -263,10 +295,11 @@ daos_csummer_init(struct daos_csummer **obj, struct csum_ft *ft,
 
 int
 daos_csummer_type_init(struct daos_csummer **obj, enum DAOS_CSUM_TYPE type,
-		       size_t chunk_bytes, bool srv_verify)
+		       size_t chunk_bytes, bool srv_verify, bool dedup,
+		       bool dedup_verify, size_t dedup_bytes)
 {
 	return daos_csummer_init(obj, daos_csum_type2algo(type), chunk_bytes,
-				 srv_verify);
+				 srv_verify, dedup, dedup_verify, dedup_bytes);
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?